tests/kuttl/common/assert_sample_deployment.yaml

```diff
@@ -43,10 +43,6 @@ spec:
       memory: 500Mi
   secret: osp-secret
 status:
-  apiEndpoint:
-    admin: http://keystone-admin-openstack.apps-crc.testing
-    internal: http://keystone-internal-openstack.apps-crc.testing
-    public: http://keystone-public-openstack.apps-crc.testing
```

Comment:
Unfortunately asserting that keystone has endpoints is part of the functionality we want to check here. So while I understand that the data here is deployment specific (ie. different in CRC than in SNO) simply not asserting such data feels wrong to me too. Unfortunately I cannot offer a solution here. Kuttl's assert possibilities seems limiting here.

Reply:
like run a command which uses a regexp?

Reply:
We could check with some script, it'll be less clean, but should be doable. Let me try and report back

Reply:
Yeah having a check that we have all the endpoints with some URL but ignoring the actual value of the URL would be better. I don't like having all these moving towards bash scripts as asserts but I don't see another option right now.